.github/workflows/pre-submit.actions.yml

```diff
@@ -9,7 +9,7 @@ permissions: read-all
 
 jobs:
   checkout:
-    name: verify no checkout in Actions
```

Comment:
If the name of a job changes then the protected branch's required pre-submits need to change as well. Ideally GitHub would allow you to set required pre-submits by an ID rather than name but it seems to use the name. We should think of job names as IDs.

I'm assuming that you're changing the name here just to add capitalization for consistency's sake? It's kind of hard keeping track of which of these are required for pre-submits so I suggest reverting the names and adding a comment that they shouldn't be changed unless there is a really good reason.